src/app/static/src/js/components/runReport/runningReportDetails.vue

```diff
@@ -0,0 +1,180 @@
+                            <span>Status:</span>
+                            <span class="font-weight-bold">{{ reportLog.status }}</span>
+                        </div>
+                    </div>
+                    <div id="report-version" v-if="reportLog.report_version" class="col-sm-auto">
+                        <div class="text-right">
+                            <span>Report version:</span>
+                            <span class="font-weight-bold"><a :href="versionUrl">{{reportLog.report_version}}</a></span>
+                        </div>
+                    </div>
+                </div>
+                <div id="report-logs" class="row pt-2">
+                    <div class="text-right col-sm-10">
+                        <textarea class="form-control bg-white"
+                                  readonly rows="10">{{ reportLog.logs }}
+                        </textarea>
+                    </div>
+                </div>
+            </div>
+            <error-info :default-message="defaultMessage" :api-error="error"></error-info>
+        </div>
+        <div id="no-logs" v-if="!reportLogSize">There are no logs to display</div>
+    </div>
+</template>
+
+<script lang="ts">
+    import Vue from "vue"
+    import {ReportLog} from "../../utils/types";
+    import {api} from "../../utils/api";
+    import ErrorInfo from "../errorInfo.vue";
+
+    interface Methods {
+        getLogs: () => void
+    }
+
+    interface Data {
+        reportLog: ReportLog
+        error: string,
+        defaultMessage: string
+    }
+
+    interface Computed {
+        paramSize: number
+        instanceSize: number
+        versionUrl: string
+        reportLogSize: boolean
+    }
+
+    interface Props {
+        reportKey: string
+    }
+
+    const initialReportLog = {
+        email: "",
+        date: "",
+        report: "",
+        instances: new Map,
+        params: new Map,
+        git_branch: "",
+        git_commit: "",
+        status: "",
+        logs: "",
+        report_version: ""
+    }
+
+    export default Vue.extend<Data, Methods, Computed, Props>({
+        name: "runningReportDetails",
+        props: {
+            reportKey: {
+                type: String,
+                required: true
+            }
+        },
+        components: {
+            ErrorInfo
+        },
+        data(): Data {
+            return {
+                reportLog: initialReportLog,
+                error: "",
+                defaultMessage: ""
+            }
+        },
+        computed: {
+            paramSize: function () {
+                return Object.keys(this.reportLog.params).length
+            },
+            instanceSize: function () {
+                return Object.keys(this.reportLog.instances).length
+            },
+            versionUrl: function () {
+                return `/report/${this.reportLog.report}/${this.reportLog.report_version}/`
+            },
+            reportLogSize: function () {
```

Comment:
This seems a bit convoluted - it's to check that you have some non-falsy values in your reportLog object, therefore it is actually a real report log? And this is because you're using the empty `initialReportLog` object? Might be simpler just to check the `name` of the report log is empty? But actually, it would be closer to the pattern we tent to use elsewhere for the type of reportLog to be `ReportLog | null`, initialise it to null and use `v-if` on the element displaying log properties to not show anything if the reportLog is null.